lib/matplotlib/transforms.py

```diff
@@ -2680,6 +2680,28 @@ def get_matrix(self):
         return self._mtx
 
 
+class _VectorTransform(Affine2DBase):
+    """
+    A transform wrapper that ignores the offset component of a base transform.
+    """
+
+    def __init__(self, transform, **kwargs):
+        super().__init__(**kwargs)
+        self._base_transform = transform
+
+    def __str__(self):
+        return ("{}(\n"
+                    "{})"
+                .format(type(self).__name__,
+                        _indent_str(self._base_transform)))
```

Comment:
Is there a reason for this 4-line construct? Would this not be the same as 

        return "{}(\n{})".format(type(self).__name__, 
                                  _indent_str(self._base_transform))

Reply:
This is consistent with how all other reprs are defined in this module (i.e. the format string's layout matches the final result).

Reply:
this was rewritten using _make_str_method.